### ISSUE TITLE
Kards Portable for Windows

### ISSUE THREAD

**Issue by JayPiEs** [Alteka/Kards#104]
Using Kards on diverse laptops on events, it would be great to have a portable version, to not always install software on client laptops.

**Comment by DrewLX** [comment]
Good idea. For 1.4 we want to improve install and roll out options, so we will take this into consideration.
